flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js

```diff
@@ -248,4 +287,101 @@ class CDUWindPage {
             speed: speed
         };
     }
+
+    static WindRequest(mcdu, stage, _showPage) {
+        getSimBriefOfp(mcdu, () => {}, () => {
+            switch (stage) {
+                case "CLB":
+                    const clbWpts = mcdu.simbrief.navlog.filter((val) => val.stage === stage);
+                    if (clbWpts[clbWpts.length - 1].ident == "TOC") {
+                        clbWpts.pop(); // Remove TOC from CLB list
+                    }
+
+                    // iterate through each clbWpt grabbing the wind data
+                    const windData = [];
+                    let lastAltitude = 0;
+                    clbWpts.forEach((clbWpt, wptIdx) => {
```

Comment:
How come we add climb winds by waypoint, but not the descent winds. Should this not be symmetrical? From my understanding, the Airbus only splits up wind predictions by waypoint for the cruise phase.

Reply:
I mentioned in the additional context that this might be desirable. Exactly the kind of feedback I was hoping for. 

> It's possible we want to extend the logic of closest altitude to descent across the STAR waypoints if we find it works well

Mostly this is due to the fact that the wind data in json form from Simbrief doesn't have the Climb sand Descent data like their generated PDF, just a set of waypoints, TOC and TOD. I see now that this needs to be applied to descent properly. I'll get that fixed up.